overlord/snapstate/snapmgr_test.go

```diff
@@ -698,7 +698,9 @@ func (s *snapmgrTestSuite) TestRemoveIntegration(c *C) {
 
 	// verify snapSetup info
 	tasks := ts.Tasks()
-	task := tasks[len(tasks)-1]
+	// XXX -2 because the last task is discard-conns and snap-setup is in the
+	// discard-snap, just before discard-conns.
```

Comment:
Let's please use this here:

`// snap-setup is in discard-snap above discard-conns.`